go-controller/pkg/ovn/egressip.go

```diff
@@ -611,15 +619,27 @@ func (oc *Controller) addEgressNode(egressNode *kapi.Node) error {
 
 func (oc *Controller) deleteEgressNode(egressNode *kapi.Node) error {
 	klog.V(5).Infof("Egress node: %s about to be removed", egressNode.Name)
-	if stdout, stderr, err := util.RunOVNNbctl(
-		"remove",
-		"logical_switch_port",
-		types.EXTSwitchToGWRouterPrefix+types.GWRouterPrefix+egressNode.Name,
-		"options",
-		"nat-addresses=router",
-	); err != nil {
-		klog.Errorf("Unable to remove GARP configuration on external logical switch port for egress node: %s, stdout: %s, stderr: %s, err: %v", egressNode.Name, stdout, stderr, err)
+	// TODO: migrate this code to the operations model once there is a good way to delete items from maps
```

Comment:
@JacobTanenbaum If you pass in a model with the OnModelMutations using map where the data contains the default value ("" in this case), you can simply use Update and that key will be removed. How about we do that and then you no longer have this as a TODO?

```
	lsp := nbdb.LogicalSwitchPort{
		Name:    types.EXTSwitchToGWRouterPrefix + types.GWRouterPrefix + egressNode.Name,
		Options: map[string]string{"nat-addresses": ""},
	}
	opModel := libovsdbops.OperationModel{
		Model: &lsp,
		OnModelMutations: []interface{}{
			&lsp.Options,
		},
		ErrNotFound: true,
	}
	if _, err := oc.modelClient.Update(opModel); err != nil {
```

Reply:
Ref code:  https://github.com/ovn-org/ovn-kubernetes/blob/4a8545134ebad337438490dbbc91072fa2e5321d/go-controller/pkg/ovn/libovsdbops/model_client.go#L193-L194

Reply:
done